app/decorators/material_decorator.rb

```diff
@@ -20,6 +20,7 @@ def decorated
       m.waste = rma['waste']
       m.coefficient = rma['coefficient']
       m.container = reaction_material.sample.container
+      m.has_scan_task = reaction_material.sample.scan_task?
```

Comment:
The material decorator is obsolete and should not be used anymore (I forgot to remove it when doing the conversion from ActiveModel::Serializer to Grape::Entity).

As the field is already included in the SampleEntity, no further action is required, as all ReactionMaterial-Entities include SampleEntity (see https://github.com/ComPlat/chemotion_ELN/blob/development-5/app/api/entities/reaction_material_entity.rb#L13)